src/app/components/pages/game-active/game-active.component.html

```diff
@@ -1,36 +1,35 @@
 <div class="container">
-  <h1>Placeholder!</h1>
+  <h1>Du bist an der Reihe: {{currentPlayerName}}!</h1>
 </div>
 
+<div class="container">
+
+
+</div>
 <div class="container">
   <app-leaderboard></app-leaderboard>
   <app-dartboard></app-dartboard>
+
 </div>
 
-<div class="container">
+<div class="playerScore container">
```

Comment:
Don't do empty spaces in class names

Reply:
For a html component to have multiple classes you seperate the names of the classes with spaces. For reverence: https://www.w3schools.com/html/html_classes.asp

Reply:
Oh sorry. Thanks for the hint :)